migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -159,6 +161,52 @@ This section will list the changes that will impact more than one component
 
         </details>
 
+### **_Update to Grid Breakpoint Naming Standard_**
+
+[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)
+
+-   **Description of change:** The grid breakpoint naming convention has been updated to a more consistent numerical system. The old breakpoint names like xxs and xxl have been replaced with 2xs, 2xl, and so on, to better align with the numeric sizing system.
+
+-   **Reason for change:** The new naming convention standardizes how breakpoints are referenced across the design system.
+
+-   **Components impacted:**
+
+    Any component or layout that references grid breakpoints using the previous naming conventions will need to be updated
+
+-   **Migration steps:**
+
+    1. Locate all instances of grid breakpoints in your codebase.
+    2. Replace old breakpoint names with the new corresponding names:
```

Comment:
I think the wording of this is a bit more clear

### **_Standardisation of breakpoint size naming_**

[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)

-   **Description of change:** The breakpoint naming convention has been updated to match other size naming like the padding and margin sizes. The old breakpoint size names like xxs and xxl have been replaced with 2xs, 2xl, and so on, to better align with other size naming.

-   **Reason for change:** The new naming convention standardises how breakpoints are referenced across the design system.

-   **Components impacted:**

    Any component or layout that references breakpoints using the previous naming convention will need to be updated.

-   **Migration steps:**

    1. Locate all instances of breakpoints in your codebase.
    2. Replace old breakpoint names with the new corresponding names: